api/src/wfl/service/rawls.clj

```diff
@@ -88,3 +112,50 @@
                     :body         (json/write-str (map make-request entities)
                                                   :escape-slash false)})
         util/response-body-json)))
+
+;; https://cromwell.readthedocs.io/en/stable/execution/ExecutionTwists/
+;; :NoNewCalls is the default.
+;;
+(def a-rawls-submission
```

Comment:
Is this used anywhere?  What is its purpose?

Reply:
To remind me what a Rawls submission looks like.